src/foam/u2/Element2.js

```diff
@@ -435,22 +435,17 @@ foam.CLASS({
       flags: ['js'],
       factory: function() { return foam.u2.DefaultValidator.create(); }
     },
-    {
-      documentation: `Keys which respond to keydown but not keypress`,
-      name: 'KEYPRESS_CODES',
-      value: { 8: true, 13: true, 27: true, 33: true, 34: true, 37: true, 38: true, 39: true, 40: true }
-    },
     {
       name: 'NAMED_CODES',
       value: {
         '8':   'backspace',
         '13':  'enter',
-        '27':  'esc',
-        '37':  'left',
-        '38':  'up',
-        '39':  'right',
-        '40':  'down',
-        '127': 'del'
+        '27':  'escape',
```

Comment:
I agree with the change, but won't this break some existing code? Also, won't the same change need to be made in U2?

Reply:
I dont see any instances where this would break existing code. I am choosing not to make this change in U2 since the console wont work in U2 anyway and we seem to be really close to switching over to U3